runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -384,6 +385,7 @@ private GroundItem buildGroundItem(final Tile tile, final TileItem item)
 		final int realItemId = itemComposition.getNote() != -1 ? itemComposition.getLinkedNoteId() : itemId;
 		final int alchPrice = itemComposition.getHaPrice();
 		final boolean dropped = tile.getWorldLocation().equals(client.getLocalPlayer().getWorldLocation()) && droppedItemQueue.remove(itemId);
+		final boolean onTable = tableItemQueue.isEmpty();
```

Comment:
This seems wrong. If two items spawn on the same tick, this doesn't make any attempt to check the item ID, tile height, or location to ensure it is the item we placed on a table.

Reply:
I think checking for itemID should be enough.